app/helpers/ems_storage_helper/textual_summary.rb

```diff
@@ -15,7 +15,7 @@ def textual_group_relationships
                        parent_ems_cloud cloud_volumes cloud_volume_snapshots cloud_volume_backups
                        cloud_object_store_containers custom_button_events physical_storages storage_resources
       ]
-    relationships.push(:cloud_volume_types) if @record.kind_of?(ManageIQ::Providers::StorageManager::CinderManager)
+    relationships.push(:cloud_volume_types) if @record.kind_of?(ManageIQ::Providers::Openstack::StorageManager::CinderManager)
```

Comment:
Ah actually now that I see that spec failure this line make sense now.  The ems_cinder factory was a `ManageIQ::Providers::StorageManager::CinderManager` type.  Only the Openstack::Cinder subclass has `cloud_volume_types` so this was a hack to "make it work"

I bet I can remove this line now

Reply:
Nope no such luck, disregard :smile: this just depends on the FactoryBot change to ems_cinder in core